### ISSUE TITLE
[Bug]: alignment step is happening after the basecalling of the first chunk

### ISSUE THREAD

**Issue by tkik** [epi2me-labs/wf-basecalling#4]
### What happened?

Thank you for implementing the workflow. 
I was using a previous version of this tool and it run without a problem. I needed to re-do the analysis using a different reference, but I realized that the resulted cram file is much smaller than it was previously. I checked the timeline and it seems that the alignment happened after finishing only the first chuck of the basecalling and the next chunks are not taken into account by the alignment step. The pipeline finishes without indicating an error. See the trace.txt for details. 

### Operating System

ubuntu 20.04

### Workflow Execution

Command line

### Workflow Execution - CLI Execution Profile

Singularity

### Workflow Version

wf-basecalling v0.4.0-gab52f81

### Relevant log output

```shell
task_id	hash	native_id	name	status	exit	submit	duration	realtime	%cpu	peak_rss	peak_vmem	rchar	wchar
3	80/869424	45440	getVersions	COMPLETED	0	2023-02-22 16:11:55.505	3.8s	1.9s	93.9%	166.1 MB	2.4 GB	134.3 KB	323 B
2	54/2e4c64	45441	getParams	COMPLETED	0	2023-02-22 16:11:55.564	3.9s	11ms	46.1%	0	0	75.6 KB	1.9 KB
1	86/7c122d	45442	wf_dorado:make_mmi (1)	COMPLETED	0	2023-02-22 16:11:55.613	1m 49s	1m 43s	170.7%	11.6 GB	15.3 GB	3 GB	7 GB
4	b8/eb1d49	45443	wf_dorado:dorado (1)	COMPLETED	0	2023-02-22 16:11:55.684	23m 19s	23m 16s	123.8%	3.9 GB	28.1 GB	9.5 GB	2.6 GB
7	fa/81c545	45445	wf_dorado:align_and_qsFilter (1)	COMPLETED	0	2023-02-22 16:35:15.585	9m 19s	9m 13s	565.2%	13.3 GB	15.5 GB	27.1 GB	5 GB
9	28/2fe79a	45447	wf_dorado:merge_fail_calls (1)	COMPLETED	0	2023-02-22 16:44:35.506	58.9s	56.4s	67.0%	858.2 MB	986.3 MB	8.9 GB	4.4 MB
8	f3/07aa16	45446	wf_dorado:merge_pass_calls (1)	COMPLETED	0	2023-02-22 16:44:35.456	1m 24s	1m 21s	81.5%	1014.7 MB	1.1 GB	6.8 GB	472.4 MB
10	79/2eff89	45448	output (1)	COMPLETED	0	2023-02-22 16:46:00.448	3.9s	5ms	83.1%	0	0	73.9 KB	252 B
11	4b/081354	45449	output (2)	COMPLETED	0	2023-02-22 16:46:00.490	3.9s	6ms	52.3%	0	0	73.9 KB	252 B
13	7a/89cdf3	45450	output (4)	COMPLETED	0	2023-02-22 16:46:00.528	3.9s	7ms	60.8%	0	0	73.9 KB	252 B
12	8a/fd8c15	45451	output (3)	COMPLETED	0	2023-02-22 16:46:00.565	3.9s	5ms	75.9%	0	0	73.9 KB	252 B
14	31/decff9	45452	output (5)	COMPLETED	0	2023-02-22 16:46:00.612	3.9s	6ms	52.1%	0	0	73.9 KB	252 B
15	da/421afd	45453	output (6)	COMPLETED	0	2023-02-22 16:46:00.649	3.9s	6ms	50.0%	0	0	73.9 KB	252 B
5	9e/3de060	45444	wf_dorado:dorado (2)	COMPLETED	0	2023-02-22 16:35:14.536	27m 45s	27m 41s	111.2%	3.7 GB	20.5 GB	9.5 GB	2.6 GB
6	8b/db738a	45454	wf_dorado:dorado (3)	COMPLETED	0	2023-02-22 17:02:59.540	30m 40s	30m 34s	111.6%	3.9 GB	28.2 GB	9.8 GB	2.6 GB
```

**Comment by SamStudio8** [comment]
@tkik Thanks for reporting this. Fixing ASAP, stand by!

**Comment by SamStudio8** [comment]
Hi @tkik, this should be fixed in wf-basecalling 0.4.1, please let me know how you get on. If you use Nextflow to manage the workflow version you can update with `nextflow pull epi2me-labs/wf-basecalling`.

**Comment by tkik** [comment]
Works great, thank you for the quick fix!
